unit-tests/live/memory/test-extrinsics.cpp

```diff
@@ -2,7 +2,8 @@
 // Copyright(c) 2021 Intel Corporation. All Rights Reserved.
 
 //#cmake: static!
-//#test:device D400*
+//#test:device D465
```

Comment:
Evgeni and I just removed the D465 from LibCI -- so just leave D435...

Reply:
fixed